src/Maestro/DependencyUpdater/DependencyUpdater.cs

```diff
@@ -146,13 +146,39 @@ public async Task<TimeSpan> RunAsync(CancellationToken cancellationToken)
         [CronSchedule("0 0 5 1/1 * ? *", TimeZones.PST)]
         public async Task CheckDailySubscriptionsAsync(CancellationToken cancellationToken)
         {
-            using (Logger.BeginScope(
-                "Updating daily subscriptions"))
+            await CheckSubscriptionsAsync(UpdateFrequency.EveryDay, cancellationToken);
+        }
+        
+        /// <summary>
+        ///     Check "TwiceDaily" subscriptions at 5 AM and 7 PM
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        [CronSchedule("0 0 5,19 * * ?", TimeZones.PST)]
```

Comment:
You'll also need to add new ProcessFinishedReleasesAsync versions (see other CronSchedule attribute usage in the release pipeline runner)

Reply:
ProcessFinishedReleasesAsync uses Cron, but I don't think it applies to this change. ProcessFinishedReleasesAsync is scheduled to run every 5 minutes to check for finished release pipelines.

Reply:
Oh oh you're right there.